contents/blog/posthog-vs-logrocket.md

```diff
@@ -0,0 +1,775 @@
+            <td>Sync event and person data</td>
+        </tr>
+        <tr>
+            <td><strong>Salesforce</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Sync event and person data</td>
+        </tr>
+        </tr>
+        <tr>
+            <td><strong>Zapier</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Export events for use in Zaps</td>
+        </tr>
+        <tr>
+            <td><strong>Shopify</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Sync customer and order data </td>
+        </tr>
+        <tr>
+            <td><strong>Stripe</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Sync customer and invoice data</td>
+        </tr>
+        <tr>
+            <td><strong>PagerDuty</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Receive customized alerts from insights</td>
+        </tr>
+        <tr>
+            <td><strong>Intercom</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Extract and load data to external platforms</td>
+        </tr>
+        <tr>
+            <td><strong>Customer.io</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Sync data between platforms</td>
+        </tr>
+        <tr>
+            <td><strong>Sentry</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Ingest Sentry errors for analysis</td>
+        </tr>
+        <tr>
+            <td><strong>Segment</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Ingest events from Segment</td>
+        </tr>
+    </tbody>
+</table>
+</div>
```

Comment:
This table could really do with some categorization. It's a bit large and hard to parse atm.

Reply:
Just to repeat the above, but this table is way to big to not have categories / some organisation – i.e. have a section for CDPs, a section for marketing integrations etc. It doesn't have to separate tables to do this, though.